### PR TITLE
Update 200-one-to-many-relations.mdx

### PR DESCRIPTION
**Pull request by sliepe**
## Describe this PR

- Wrong description for a following example

## Changes

- Corrected description for a following example

## What issue does this fix?

- N/A

## Any other relevant information

- https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-many-relations#mongodb
  - ~~Wrong~~: "The following list of Post MongoDB documents each have a ~~userId~~ field which reference the same user:"
  - **Right**: "The following list of Post MongoDB documents each have a **authorId** field which reference the same user:"

### CODE REVIEW

**Review by sliepe — COMMENTED** [review]
Simply approve please

File: content/200-concepts/100-components/01-prisma-schema/06-relations/200-one-to-many-relations.mdx [hunk 1]
```diff
@@ -179,7 +179,7 @@ The following MongoDB document represents a `User`:
 { "_id": { "$oid": "60d5922d00581b8f0062e3a8" }, "name": "Ella" }
 ```
 
-The following list of `Post` MongoDB documents each have a `userId` field which reference the same user:
+The following list of `Post` MongoDB documents each have a `authorId` field which reference the same user:
```
Comment (zachtil) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/200-one-to-many-relations.mdx]:
```suggestion
Each of the following `Post` MongoDB documents has an `authorId` field which references the same user:
```

**Review by zachtil — CHANGES_REQUESTED** [review]

File: content/200-concepts/100-components/01-prisma-schema/06-relations/200-one-to-many-relations.mdx (hunk 1 above)
Comment (sliepe) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/200-one-to-many-relations.mdx]: Sorry for too late.

**Comment by nikolasburk** [comment]
This makes sense, thanks for caring about our documentation @sliepe 🙏 💚

**Pull request merged**